charts/bulk-scan-orchestrator/values.template.yaml

```diff
@@ -0,0 +1,34 @@
+java:
+  applicationPort: 8582
+  readinessPath: '/health'
+  livenessPath: '/health'
+  secrets:
+    SB_CONN_STRING:
+      secretRef: servicebus-secret-namespace-${SERVICE_NAME}
+      key: connectionString
+  environment:
+    ENVELOPES_QUEUE_CONNECTION_STRING: "$(SB_CONN_STRING);EntityPath=envelopes"
+    PROCESSED_ENVELOPES_QUEUE_CONNECTION_STRING: "$(SB_CONN_STRING);EntityPath=processed-envelopes"
+  keyVaults:
+    "bulk-scan":
+      resourceGroup: bulk-scan
+      secrets:
+        - idam-client-secret
+        - idam-users-bulkscan-username
+        - idam-users-bulkscan-password
+        - s2s-secret-for-tests
```

Comment:
this shouldn't be needed for the application to be run,
it should be:
`microservicekey-bulk-scan-orchestrator`
from the s2s vault

https://github.com/hmcts/bulk-scan-orchestrator/blob/master/infrastructure/main.tf#L112